react-app/src/app/screens/User/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { NavLink, Redirect, Route, Switch } from 'react-router-dom';
+
+import style from './styles.scss';
+import Dashboard from './components/Dashboard';
+import History from './components/History';
+import Points from './components/Points';
+
+class Userpage extends Component {
+  render() {
+    return (
+      <div className={style.loginLayout}>
+        <h2>User page</h2>
+        <div className={style.userbarNav}>
+          <NavLink to="/user" exact className={style.userbarLink} activeClassName={style.userbarActive}>
+            Dashboard
+          </NavLink>
+          <NavLink
+            to="/user/points"
+            exact
+            className={style.userbarLink}
+            activeClassName={style.userbarActive}
+          >
+            Points
+          </NavLink>
+          <NavLink
+            to="/user/history"
+            exact
+            className={style.userbarLink}
+            activeClassName={style.userbarActive}
+          >
+            History
+          </NavLink>
+        </div>
+        <Switch>
```

Comment:
I would keep a single routing switch in the app. Otherwise, it would be pretty confusing to debug the routes, since they would be scattered throughout the app. Also, i would move every route to constants, in order to keep a single source file with routes